torch/csrc/jit/ir.cpp

```diff
@@ -1294,7 +1295,7 @@ Value* Graph::insert(
     Symbol opname,
```

Comment:
Why is it ok to pass Symbol by value but not other types? What feature of a type causes this error to occur? What if we change Symbol to no longer have this feature?

Reply:
Because clant-tidy detects that it is a trivial type (`uint32_t`)

Reply:
Symbol is a struct. Is it checking a certain property on the struct?